schema/NationalPrisonerStatsSchema.mcf

```diff
@@ -0,0 +1,163 @@
+Node: dcid:Incarcerated
+name: "Incarcerated"
+typeOf: dcs:InstitutionalizationEnum
+description: "Indicates the state of being confined in a prison or a jail. This may also include halfway houses, boot camps, weekend programs, and other facilities in which individuals are locked up overnight."
+descriptionUrl: "https://www.bjs.gov/index.cfm?tv=tdtp&tid=1"
+
+Node: dcid:prisonSentenceStatus
+name: "prisonSentenceStatus"
+typeOf: schema:Property
+rangeIncludes: dcs:PrisonSentenceStatusEnum
+domainIncludes: dcs:Person,dcs:IncarcerationEvent
+
+Node: dcid:PrisonSentenceStatusEnum
+name: "PrisonSentenceStatusEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "An enumeration of prison sentence statuses."
+
+Node: dcid:Unsentenced
+name: "Unsentenced"
+typeOf: dcs:PrisonSentenceStatusEnum
+description: "Indicates state of not having received a prison sentence."
+
+Node: dcid:Sentenced
+name: "Sentenced"
+typeOf: dcs:PrisonSentenceStatusEnum
+description: "Indicates state of having received a prison sentence."
+
+Node: dcid:maxPrisonSentence
+name: "maxPrisonSentence"
+typeOf: schema:Property
+rangeIncludes: dcs:QuantityRange
+domainIncludes: dcs:Person,dcs:IncarcerationEvent
+
+Node: dcid:PrisonerMeasurementBasisEnum
+name: "PrisonerMeasurementBasisEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "Enumeration for the basis of measurement of the prisoner statistic."
+
+Node: dcid:NPS_MeasuredBasedOnCustody
+name: "NPS_MeasuredBasedOnCustody"
+typeOf: dcs:PrisonerMeasurementBasisEnum
+description: "Measurement based on those in custody of state or federal correctional officials."
+
+Node: dcid:NPS_MeasuredBasedOnJurisdiction
+name: "NPS_MeasuredBasedOnJurisdiction"
+typeOf: dcs:PrisonerMeasurementBasisEnum
+description: "Measurement based on those under the jurisdiction of the state meaning all state and federal inmates held in a public or private prison and those held in jail facilities either physically located inside or outside of the state of legal responsibility, and other inmates who may be temporarily out to court or in transit from the jurisdiction of legal authority to the custody of a confinement facility outside that jurisdiction."
+descriptionUrl: “https://www.bjs.gov/index.cfm?ty=tdtp&tid=1”
+
+Node: dcid:CauseOfDeathEnum
+name: "CauseOfDeathEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "Enumeration for causes of death."
+
+Node: dcid:JudicialExecution
+name: "JudicialExecution"
+typeOf: dcs:CauseOfDeathEnum
+description: "Execution for committing a crime."
+
+Node: dcid:IllnessOrNaturalCause
+name: "IllnessOrNaturalCause"
+typeOf: dcs:CauseOfDeathEnum
+description: "Illness or other natural cause of death excluding AIDS."
+
+Node: dcid:AIDS
+name: "AIDS"
+typeOf: dcs:CauseOfDeathEnum
```

Comment:
I wonder if we should use this? https://icd.codes/icd10cm/B20#:~:text=Codhttps://datacommons.org/browser/ICD10/B20

Reply:
Are you able to find it? We had previous thread on it https://screenshot.googleplex.com/AKTzdqu4kBNAhEK

Reply:
Ah, they've been marked outdated and don't show up on the /files page. Responded on there.

Reply:
I just realized, the one we have might actually include aids because it says it corresponds to codes B20-B24, should I use this instead? https://screenshot.googleplex.com/YnaZm8nTRPL62gw.png

Reply:
I would prefer to use this: https://datacommons.org/browser/ICD10/B20 but we can fix this later too.